mkdocs.yml

```diff
@@ -88,6 +88,7 @@ nav:
           - "guides/guide-dev/index.md"
       - Administrators:
           - "guides/guide-admin/index.md"
```

Comment:
I think it would be good to add a reference to the new page to the admin docs index page.

Also, see in the "Contributors" section below how you can give the subpages of each section different. It would probably be good to do that as well to make the index on the left side look pretty.